link-paths.py

```diff
@@ -117,7 +142,15 @@ def main() -> None:
                 alt=run["mean_altitude"] * u.rad,
                 az=run["mean_azimuth"] * u.rad,
             )
-            nearest_irf = pointing.separation(irf_pointings).argmin()
+            sindelta = sin_delta(pointing)
```

Comment:
`sin_delta` to be consistent with above?

Reply:
Nah, that would overwrite the function. But you are right, I am not happy with the naming here.

Reply:
Change the other ones for consistency.